GraphNotifications/GraphNotificationsSample/APNSToken.h

```diff
@@ -0,0 +1,18 @@
+//
+//  APNSToken.h
+//  GraphNotificationsSample
+//
+//  Created by Adam Crabtree on 12/27/19.
+//  Copyright © 2019 Microsoft. All rights reserved.
```

Comment:
We use the [MIT X11 license](LICENSE) for this repo:
```suggestion
//  Copyright (c) Microsoft. Licensed under the MIT license.
```

Reply:
Similar comments apply to other copyright comments.

Reply:
This has been resolved.

Reply:
I still see "All rights reserved" across many files; please check again.

Reply:
Oooh whoops. Sorry I didn't understand. This has now been fixed.